### PR TITLE
 new: runtime tracepoints management 

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

/area driver-kmod

/area driver-bpf

/area driver-modern-bpf

/area libscap-engine-bpf

/area libscap-engine-kmod

/area libscap-engine-modern-bpf

/area libscap

/area libsinsp

**Does this PR require a change in the driver versions?**

/version driver-API-version-major

new ioctl introduced in kmod.

**What this PR does / why we need it**:

This PR completes part of #521  allowing even tracepoints to be runtime modified, just like we allow it for ppm_sc.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
Rebased on top of master.

**Comment by FedeDP** [comment]
Rebased on top of master!

File: userspace/libsinsp/sinsp.h [hunk 1]
```diff
@@ -922,6 +922,18 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 
 	/*=============================== Tracepoint set related ===============================*/
 
+	/*!
+		\brief Mark desired tracepoint as (un)interesting, attaching or detaching it.
+		This method receives a `tp` code. You can find the available
+		`enum ppm_syscall_code` in `driver/ppm_tp.h`.
```
Comment (Andreagit97) [inline userspace/libsinsp/sinsp.h:928]:
```suggestion
		`enum tp_values` in `driver/ppm_tp.h`.
```

**Review by Andreagit97 — COMMENTED** [review]
LGTM

**Review by leogr — APPROVED** [review]

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**